compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -415,7 +416,9 @@ void OMR::Power::MemoryReference::addToOffset(TR::Node * node, int64_t amount, T
       self()->setBaseModifiable();
```

Comment:
Also, more efficient handling of big (over 32bit) displacement should be put in place for P10 and later (e.g. using prefixed instructions).

Reply:
Not sure exactly what you mean here. The code above is already using `loadActualConstant` when trying to accumulate large offsets, so it should use whatever handling all other constants use in that case, which would include using `ConstantDataSnippet` loads when loading the out-of-range parts of these displacements during accumulation. Can you please clarify what you mean?

Reply:
mis-reading the order of changes ... it looks ok, although more optimal code is possible and desired (for p10 or pTOC).